R/db.R

```diff
@@ -175,7 +175,8 @@ verify_table <- function(con, table_name, table, additional_columns = NULL,
 #' @keywords internal
 parse_sql_date <- function(con, date) {
   if (sql_dialect(con) == "sqlite") {
-    date <- as.POSIXct(date, origin = "1970-01-01", tz = "UTC")
+    date <- format(as.POSIXct(date, origin = "1970-01-01", tz = "UTC"),
+                   digits = 0)
```

Comment:
Also unrelated change, on devl ubuntu tests which check this were failing because the time was coming through as e.g. "16:34:23.12312354532424" where the test was not expecting any decimal points. This explicitly formats to remove any trailing digits as I think they are just noise from conversion anyway.